litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/index.tsx

```diff
@@ -146,7 +146,11 @@ const WorkflowSettings = forwardRef((_, ref) => {
       CRDLink,
     };
     localforage.setItem('workflow', workflowDetails);
-    if (validateWorkflowName(name)) {
+    if (name.length === 0) {
+      alert.changeAlertState(true); // Workflow Name is empty and user clicked on Next
+      return false;
+    }
+    if (name.length > 0 && validateWorkflowName(name)) {
```

Comment:
`name.length && validateWorkflowName(name)` can be used

Reply:
Same reason here as well it'll check for `if(0 or the length && true or false based on validation) -> if(number && boolean){}` which is not working thus the assertion is required.

Reply:
PTAL at the above one if its missing any edge case. If still it does not work we can go ahead with this version.

Reply:
name.length will return either a 0 (false value) or a non zero number (true value)...so `name.length` can be used

Reply:
I tried using `name.length` it didn't work, when the value is 0 the if condition is if(0){} which is completely skipping the if statement

Reply:
![if2](https://user-images.githubusercontent.com/34975209/113705402-02087b00-96fb-11eb-83c6-712645bf2a9c.png)

Reply:
0 is a false value and && is used...so a false value will short circuit the condition and if won't run...which should be the same as when length is < 0...either way it's fine by me.